event/monitoring.go

```diff
@@ -45,6 +45,31 @@ type CommandFailedEvent struct {
 }
 
 // CommandMonitor represents a monitor that is triggered for different events.
+//
+// For an example, the following code will match up started events with their
```

Comment:
Should we remove the changes from this file if this example is now stored in `examples_test.go`?

Reply:
oops, done